system/manager/manager.py

```diff
@@ -46,7 +46,7 @@ def manager_init() -> None:
     ("DynamicExperimentalControl", "0"),
     ("Mads", "1"),
     ("MadsMainCruiseAllowed", "1"),
-    ("MadsPauseLateralOnBrake", "0"),
+    ("MadsSteeringMode", "1"),
```

Comment:
If you do https://github.com/sunnypilot/sunnypilot/pull/687/files#r2004382635, you avoid the value change here

```suggestion
    ("MadsSteeringMode", "0"),
```